webdriver-spec.html

```diff
@@ -3753,19 +3753,14 @@
  with the string "<code>element-6066-11e4-a52e-4f735466cecf</code>".
 
 <p>Each <a>element</a> has an associated <dfn>web element
- reference</dfn> that uniquely identifies the <a>element</a> across
- all <a>browsing contexts</a>.  The <a>web element reference</a> for
- every <a>element</a> representing the same <a>element</a> is the
- same. The <a>web element reference</a> should be a <a>UUID</a>.
+ reference</dfn> (a <a>UUID</a>)
```

Comment:
There's no requirement for the reference to be a UUID. See 81bd8355e6c509c8349f2e7302960148b5a95703. Use `should` to make this clear